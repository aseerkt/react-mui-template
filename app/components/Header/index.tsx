const Header = () => {
  return (
    <header>
      <nav>
        <h1>Poke React</h1>
      </nav>
    </header>
  );
};

export default Header;
