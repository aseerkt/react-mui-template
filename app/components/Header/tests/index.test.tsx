import { render } from '@testing-library/react';
import Header from '..';

describe('<Header /> tests', () => {
  it('should match snapshot', () => {
    const { baseElement } = render(<Header />);
    expect(baseElement).toMatchSnapshot();
  });
});
