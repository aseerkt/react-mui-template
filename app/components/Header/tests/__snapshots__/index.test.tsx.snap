// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> tests should match snapshot 1`] = `
<body>
  <div>
    <header>
      <nav>
        <h1>
          Poke React
        </h1>
      </nav>
    </header>
  </div>
</body>
`;
